ai-chat-assistant/autokitteh.yaml

```
@@ -0,0 +1,20 @@
+version: v1
+
+project:
+  name: ai_chatbot_assistant
+  vars:
+    - name: SHEET_ID
+      value:
+    - name: SHEET_NAME
+      value:
+    - name: REPO_NAME
+      value:
+  connections:
+    - name: chatgpt_conn
+      integration: chatgpt
+    - name: github_conn
+      integration: github
+    - name: googlesheets_conn
```

Optional nit, no special logic here, I just like this more. If you do decide to use it, obviously change the Python code too.
```suggestion
    - name: sheets_conn
```